src/main/java/screens/ListingCRUDScreen.java

```diff
@@ -16,17 +18,20 @@ public ListingCRUDScreen() {
         JButton createListing = new JButton("Create Listing");
         JButton browseListings = new JButton("Browse Listings");
         JButton removeListings = new JButton("Remove Listings");
+        JButton findNearestDealership = new JButton("Find Nearest Dealership");
         JButton logOut = new JButton("Log Out :(");
 
         buttons.add(createListing);
         buttons.add(browseListings);
         buttons.add(removeListings);
+        buttons.add(findNearestDealership);
```

Comment:
This button should only be visible when an Individual User is logged in and not a Dealership User right? I suggest making use of the singleton here. It is after all a great design pattern that we have used!

Reply:
Ah, but of course!

Reply:
I think I'll leave that part to Ryan since UserLogin is his use case!